packages/dbml-core/src/parse/ANTLR/ASTGeneration/postgres/PostgresASTGen.js

```diff
@@ -804,6 +804,29 @@ export default class PostgresASTGen extends PostgreSQLParserVisitor {
             cmd.value.endpoints[0].schemaName = schemaName;
             this.data.refs.push(cmd.value);
             break;
+          case TABLE_CONSTRAINT_KIND.UNIQUE:
+          case TABLE_CONSTRAINT_KIND.INDEX:
+            if (cmd.value.columns == 0) break;
+            if (!(cmd.value.pk || cmd.value.unique)) break;
+            // eslint-disable-next-line prefer-destructuring
+            kind = cmd.kind;
+            const table = findTable(this.data.tables, schemaName, tableName);
```

Comment:
Declaring these without scoping them will give a lot of lint error, and potentially cause conflict with same name `const` in other cases. Please help enclosed the this whole code from  line 809 to line 829 within brackets { }. Thanks